{ 'ready_state': { 'head': [ { 'duration': 3.016,
                               'name': u'ready_state',
                               'positions': [-0.0, 0.0022],
                               'starttime': 0.0}],
                   'left_arm': [ { 'duration': 3.016,
                                   'name': u'ready_state',
                                   'positions': [ -1.5135,
                                                  0.7532,
                                                  -0.2755,
                                                  0.0212,
                                                  0.0208,
                                                  -0.2499,
                                                  0.8168],
                                   'starttime': 0.0}],
                   'left_leg': [ { 'duration': 3.016,
                                   'name': u'ready_state',
                                   'positions': [ 0.0113,
                                                  -0.0746,
                                                  -0.7899,
                                                  1.1893,
                                                  0.6074,
                                                  -0.0704],
                                   'starttime': 0.0}],
                   'right_arm': [ { 'duration': 3.016,
                                    'name': u'ready_state',
                                    'positions': [ 1.5135,
                                                   -0.7532,
                                                   0.2755,
                                                   -0.0212,
                                                   -0.0208,
                                                   0.2499,
                                                   -0.8168],
                                    'starttime': 0.0}],
                   'right_leg': [ { 'duration': 3.016,
                                    'name': u'ready_state',
                                    'positions': [ -0.0113,
                                                   0.0746,
                                                   0.7899,
                                                   -1.1893,
                                                   -0.6074,
                                                   0.0704],
                                    'starttime': 0.0}],
                   'torso': [ { 'duration': 3.016,
                                'name': u'ready_state',
                                'positions': [0.0001, -0.0002],
                                'starttime': 0.0}]}}
